allofplos/corpus/plos_corpus.py

```diff
@@ -389,7 +389,7 @@ def get_uncorrected_proofs(directory=None, proof_filepath=uncorrected_proofs_tex
         print("Creating new text list of uncorrected proofs from scratch.")
         article_files = listdir_nohidden(directory)
         uncorrected_proofs = set()
-        for article_file in tqdm(article_files, disable=None):
+        for article_file in tqdm(article_files, disable=None, miniters=len(article_files)/1000):
```

Comment:
I think this will return a float sometimes… that might cause an iteration error…

Reply:
To solve this you could include

```
from math import ceil
⋮
 for article_file in tqdm(article_files, disable=None, miniters=ceil(len(article_files)/1000)):
```
if you want the ceiling, or the same replacing `ceil` with `floor` if you want the `floor`.

Reply:
Note, in the tqdm docs it [does expect it to be a float](https://pypi.python.org/pypi/tqdm#parameters):

> miniters : int, optional
Minimum progress display update interval, in iterations. If 0 and dynamic_miniters, will automatically adjust to equal mininterval (more CPU efficient, good for tight loops). If > 0, will skip display of specified number of iterations. Tweak this and mininterval to get very efficient loops. If your progress is erratic with both fast and slow iterations (network, skipping items, etc) you should set miniters=1.